Error when using `save_missing` attribute with TV show library

Works fine with movie library, but tv library throws the below error:

```
[2021-02-21 20:48:19,097] [util.py:594]               [DEBUG]    | Traceback (most recent call last):                                                                 |
                                                                 |   File "/mnt/c/repos/unraid/Plex-Meta-Manager/modules/config.py", line 401, in update_libraries
                                                                 |     builder.run_methods(collection_obj, collection_name, map, movie_map, show_map)
                                                                 |   File "/mnt/c/repos/unraid/Plex-Meta-Manager/modules/builder.py", line 658, in run_methods
                                                                 |     self.library.add_missing(c, missing_shows_with_names, False)
                                                                 | NameError: name 'c' is not defined
                                                                 | 
[2021-02-21 20:48:19,098] [config.py:413]             [ERROR]    | Unknown Error: name 'c' is not defined
```

--- a/README.md
+++ b/README.md
@@ -1,5 +1,5 @@
 # Plex Meta Manager
-#### Version 1.2.1
+#### Version 1.2.2
 
 The original concept for Plex Meta Manager is [Plex Auto Collections](https://github.com/mza921/Plex-Auto-Collections), but this is rewritten from the ground up to be able to include a scheduler, metadata edits, multiple libraries, and logging. Plex Meta Manager is a Python 3 script that can be continuously run using YAML configuration files to update on a schedule the metadata of the movies, shows, and collections in your libraries as well as automatically build collections based on various methods all detailed in the wiki. Some collection examples that the script can automatically build and update daily include Plex Based Searches like actor, genre, or studio collections or Collections based on TMDb, IMDb, Trakt, TVDb, AniDB, or MyAnimeList lists and various other services.
 
@@ -19,4 +19,5 @@ The script is designed to work with most Metadata agents including the new Plex 
 * If you have a configuration question visit the [Discussions](https://github.com/meisnate12/Plex-Meta-Manager/discussions)
 * To see user submitted Metadata configuration files and you could even add your own go to the [Plex Meta Manager Configs](https://github.com/meisnate12/Plex-Meta-Manager-Configs)
 * Pull Request are welcome but please submit them to the develop branch
+* If you wish to contribute to the Wiki please fork and send a pull request on the [Plex Meta Manager Wiki Repository](https://github.com/meisnate12/Plex-Meta-Manager-Wiki)
 * [Buy Me a Pizza](https://www.buymeacoffee.com/meisnate12)

--- a/modules/builder.py
+++ b/modules/builder.py
@@ -655,7 +655,7 @@ class CollectionBuilder:
                                 logger.error(e)
                         logger.info("{} Show{} Missing".format(len(missing_shows_with_names), "s" if len(missing_shows_with_names) > 1 else ""))
                         if self.details["save_missing"] is True:
-                            self.library.add_missing(c, missing_shows_with_names, False)
+                            self.library.add_missing(collection_name, missing_shows_with_names, False)
                         if self.do_arr and self.library.Sonarr:
                             self.library.Sonarr.add_tvdb([missing_id for title, missing_id in missing_shows_with_names], tag=self.details["arr_tag"])
 

--- a/plex_meta_manager.py
+++ b/plex_meta_manager.py
@@ -59,7 +59,7 @@ logger.info(util.get_centered_text("| |_) | |/ _ \ \/ / | |\/| |/ _ \ __/ _` | |
 logger.info(util.get_centered_text("|  __/| |  __/>  <  | |  | |  __/ || (_| | | |  | | (_| | | | | (_| | (_| |  __/ |   "))
 logger.info(util.get_centered_text("|_|   |_|\___/_/\_\ |_|  |_|\___|\__\__,_| |_|  |_|\__,_|_| |_|\__,_|\__, |\___|_|   "))
 logger.info(util.get_centered_text("                                                                     |___/           "))
-logger.info(util.get_centered_text("    Version: 1.2.1                                                                   "))
+logger.info(util.get_centered_text("    Version: 1.2.2                                                                   "))
 util.seperator()
 
 if args.tests:

